microsoft/onefuzz
# Look for azcopy.exe in AZCOPY if it's a directory

## Summary of the Pull Request

https://github.com/microsoft/onefuzz/issues/3234

## PR Checklist

## Info on Pull Request

Adds code to look for azcopy.exe inside of AZCOPY if it's a directory and prints a nice error if azcopy.exe can't be found.

## Validation Steps Performed

Set AZCOPY to a directory containing azcopy.exe and observe that commands work as expected.
Set AZCOPY to a directory _not_ containing azcopy.exe and observe that an informative error is printed out.
Set AZCOPY to a path pointing to azcopy.exe and observe that commands work as expected.
Set AZCOPY to a path pointing to an azcopy.exe that _doesn't exist_ and observe that an informative error is printed out.
